server/documents/modules/progress.html.eco

```diff
@@ -1068,6 +1068,8 @@ themes      : ['Default', 'Classic', 'Basic', 'Striped']
             error    : {
               method     : 'The method you called is not defined.',
               nonNumeric : 'Progress value is non numeric'
```

Comment:
Trailing comma is missing, also for next line.

Reply:
Oops.
Fixed in 6c6ff34